tests/test_pipeline/test_cuda_rpc_optimizer.py

```diff
@@ -0,0 +1,153 @@
+import os
+import argparse
+
+import torch
+from torch import nn
+import torch.multiprocessing as mp
+import torch.distributed.rpc as rpc
+from torch import autograd
+from torch.optim import SGD, Adam, RMSprop, Optimizer
+from colorama import Back, Style
+
+from colossalai.pipeline.rpc.PipelineBase import FillDrainPipelineEngine, OneFOneBPipelineEngine
+from colossalai.testing import assert_close
+
+
+def color_debug(text, prefix=' ', color='blue'):
+    color = color.upper()
+    print(getattr(Back, color), prefix, Style.RESET_ALL, text)
+
+
+class TestModel(nn.Module):
```

Comment:
For the maintainace, I think you could integrate this and some redundant codes below into a file named 'rpc_test_utils.py', and reuse them in rpc tests.